packages/astro/src/@types/astro.ts

```diff
@@ -746,26 +752,26 @@ export interface AstroUserConfig {
 		/**
 		 * @docs
 		 * @name image.service (Experimental)
-		 * @type {'astro/assets/services/sharp' | 'astro/assets/services/squoosh' | string}
-		 * @default `'astro/assets/services/squoosh'`
+		 * @type {{entrypoint: 'astro/assets/services/sharp' | 'astro/assets/services/squoosh' | string, config: Record<string, any>}}
+		 * @default `{entrypoint: 'astro/assets/services/squoosh', config?: {}}`
 		 * @version 2.1.0
```

Comment:
I'm just curious because I don't know: Do we need to worry about updating the version number here since this specific configuration will only be appropriate for 2.3.x? (Even though `image.service` itself was added in 2.1.0?)

Are we at all worried about this being misleading for someone who happens to be on a slightly older version of Astro before this update to the image service config?